<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="timeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>940</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Проверить время</string>
    </property>
   </widget>
   <widget class="QPushButton" name="findButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Посмотреть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Поиск по </string>
    </property>
   </widget>
   <widget class="QComboBox" name="search">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>предмету</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>олимпиаде</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="searchbox_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Статус</string>
    </property>
   </widget>
   <widget class="QLabel" name="Stage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Этап</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предмет</string>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>310</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата проведения</string>
    </property>
   </widget>
   <widget class="QComboBox" name="searchbox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="searchbox_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>360</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>40</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>1161</width>
      <height>420</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Статус</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Предмет</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Этап</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="add_check1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="add_check0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="add_check2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>360</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="selectall">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Посмотреть всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delbutton">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>940</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="datecheck">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>940</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удалить всё</string>
    </property>
   </widget>
   <zorder>edit2</zorder>
   <zorder>edit0</zorder>
   <zorder>edit1</zorder>
   <zorder>timeButton</zorder>
   <zorder>findButton</zorder>
   <zorder>label_2</zorder>
   <zorder>search</zorder>
   <zorder>searchbox_1</zorder>
   <zorder>Status</zorder>
   <zorder>Stage</zorder>
   <zorder>Name</zorder>
   <zorder>Date</zorder>
   <zorder>searchbox_2</zorder>
   <zorder>searchbox_3</zorder>
   <zorder>date</zorder>
   <zorder>addButton</zorder>
   <zorder>tableWidget</zorder>
   <zorder>add_check1</zorder>
   <zorder>add_check0</zorder>
   <zorder>add_check2</zorder>
   <zorder>selectall</zorder>
   <zorder>delbutton</zorder>
   <zorder>datecheck</zorder>
   <zorder>clear_button</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
